src/storage/documentService/documentService.js

```diff
@@ -0,0 +1,59 @@
+const verify = require("@govtechsg/oa-verify");
+const uuid = require("uuid/v4");
+const { encryptString } = require("./crypto");
+const config = require("../config");
+const { put, get, remove } = require("../dynamoDb");
+
+const DEFAULT_TTL = 1000 * 60 * 5; // 5 Minutes
+
+const putDocument = async document => {
+  const created = Date.now(); // TTL is handled by dynamoDb natively, this timestamp has to be UTC unixtime
+  // https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-how-to.html
+  const params = {
+    TableName: config.dynamodb.storageTableName,
+    Item: {
+      id: uuid(),
+      document,
+      created,
+      ttl: created + DEFAULT_TTL
+    }
+  };
+
+  return put(params).then(() => params.Item);
+};
+
+const getDocument = async (id, { cleanup }) => {
+  const params = {
+    TableName: process.env.OA_DOC_STORAGE_TABLE,
+    Key: {
+      id
+    }
+  };
+  const document = await get(params);
+  if (cleanup && cleanup === "true") {
```

Comment:
```suggestion
  if (cleanup) {
```

If it's a string, should handle at the event handler level, its just weird here.

Reply:
moved to event handler